bin/run.sh

```diff
@@ -1,3 +1,3 @@
  #! /bin/sh
 
-docker-compose -f docker/docker-compose.yml up --build && docker-compose -f docker/docker-compose.yml rm -f
+BLOCKCHAIN=eth docker-compose -f docker/docker-compose.yml up --build && docker-compose -f docker/docker-compose.yml rm -f
```

Comment:
Which part? We need the "rm -f" so that Kafka and Zookeeper containers are destroyed and not reused between restarts. And we need to provide the file name because it is not in the current directory.